### ISSUE TITLE
EAD Level 0 PARK BRAKE ON Alert not showing

### ISSUE THREAD

**Issue by omega892** [invernyx/md11-bugs#716]
### Describe the bug

Having shutdown engine number 2, if the parking brake is released and then set back to on, the PARK BRAKE ON Level 0 alert does reappear.

### Expected behaviour

Setting the parking brake to ON, I'd expect to see the PARK BRAKE ON Level 0 Alert appear in the third column on the EAD at the top. The 2 attached photos show the EAD with the parking brake off, and then with the parking brake on, at the resulting lack of alert at the top of the EAD third column.

### Evidence

![20231205102351_1](https://github.com/invernyx/md11-bugs/assets/152972533/193b9c8b-2201-4852-9676-caaa7dac6e77)
![20231205102356_1](https://github.com/invernyx/md11-bugs/assets/152972533/6a098c72-0fbe-47ae-adfa-5f2aaf9b6032)

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

Yes

### Agreement

- [X] Yes

**Comment by turbofandude** [comment]
I suspect this is due to you already have 4 L0 alerts. I wonder if it continues up the center column if there are 5?

**Comment by omega892** [comment]
I've just tried this again and still no addition to the top of the parking brake L0 alert. You can see from the 4th screenshot that the L0 alert doesn't reappear on the EAD - my understanding is that it should pop up at the top of the list? 

![20231206193209_1](https://github.com/invernyx/md11-bugs/assets/152972533/2fd02011-9b45-4ea5-9e25-574e9dded916)
![20231206193308_1](https://github.com/invernyx/md11-bugs/assets/152972533/8e2a0bfd-4009-4dcb-abf2-c3c606d55c9e)
![20231206193316_1](https://github.com/invernyx/md11-bugs/assets/152972533/516c89e6-a206-48ed-87c7-01eb353e7140)
![20231206193323_1](https://github.com/invernyx/md11-bugs/assets/152972533/02d65553-7932-4785-b660-e4159ffe5436)

**Comment by turbofandude** [comment]
Yes... this is the exact behavior as the initial bug report. I see the PARK BRAKE ON showing properly when there are 4 or less alerts - it just doesn't roll into the middle column when there are more than 4.
